src/information-for-sponsors.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Information for sponsors
+---
+
+<div class="box box_red">
+  <h3>What we need from you</h3>
+  <ul>
+    <li>the names and email addresses of the people claiming your complimentary tickets</li>
+    <li>the name, email address, and phone number of somebody who'll be at the conference, and who can act as our point of contact</li>
+    <li>a PNG/JPEG logo for the website, width 1068 pixels or larger</li>
+    <li>a URL that you'd like us to share</li>
+    <li>a short (one sentence) description of your company for the website, and a link to your website</li>
+    <li>your Twitter username if you have one</li>
+  </ul>
+</div>
+
+<div class="box box_yellow">
+  <h3>Tickets</h3>
+
+  <p>
+    All attendees, including those representing sponsors, <strong>must have their own tickets</strong>.
+  </p>
+
+  <p>
+    <a href="/sponsorship/">All sponsorship packages</a> include at least one complimentary ticket, but please ensure your package includes sufficient tickets for all your attendees. You will need to purchase additional tickets if required.
+  </p>
+
+  <p>
+    Don't hesitate to <a href="/contact/">contact us</a> if you have any questions about this.
+  </p>
+</div>
+
+<div class="box box_blue">
+  <h3>Materials</h3>
+
+  <p>
+    You may arrange to have materials delivered to the venue in advance. Please address all consignments as follows:
+  </p>
+
+  <p>
+    <strong>PyCon UK 2019</strong><br />
```

Comment:
Could we change this to say PyConUK 2022

Reply:
Hah, whoops, yes, we can do that.